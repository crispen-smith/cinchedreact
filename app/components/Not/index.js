/**
 *
 * Not
 *
 */

import styled from 'styled-components';

const Not = styled.div`
  margin: 1rem;
  text-align: center;
  font-size: 2rem;
`;

export default Not;
