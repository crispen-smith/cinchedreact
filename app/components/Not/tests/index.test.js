import React from 'react';
import renderer from 'react-test-renderer';

import Not from '../index';

describe('<Not />', () => {
  it('Renders a Not component', () => {
    const tree = renderer
      .create(<Not>This is not a what it should not be.</Not>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
