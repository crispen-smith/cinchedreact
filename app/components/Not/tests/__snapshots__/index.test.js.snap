// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Not /> Renders a Not component 1`] = `
<div
  className="Not-bJJsFZ hKjuvT"
>
  This is not a what it should not be.
</div>
`;
